lib/hydra/derivatives/processors/image_decorator.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Fix PDF tripple issue
+
+module Hydra
+  module Derivatives
+    module Processors
+      module ImageDecorator
+        protected
+
+          # When resizing images, it is necessary to flatten any layers, otherwise the background
+          # may be completely black. This happens especially with PDFs. See #110
```

Comment:
idk where this code was copied over from, but can this `#110` reference be fleshed out to point at the issue? in the event we need to reference it in the future.

Reply:
This is the issue: https://github.com/samvera/hydra-derivatives/issues/110

Reply:
```suggestion
          # may be completely black. This happens especially with PDFs. See https://github.com/samvera/hydra-derivatives/issues/110
```